app/src/ui/app.tsx

```diff
@@ -324,21 +324,17 @@ export class App extends React.Component<IAppProps, IAppState> {
         return this.props.dispatcher.showPopup({ type: PopupType.Preferences })
       case 'open-working-directory':
         return this.openCurrentRepositoryWorkingDirectory()
-      case 'update-branch': {
```

Comment:
The changes in this file are all stylistic in nature. We don't use curly brackets in switch statements anywhere else in the app (that I know of) so this change is for consistency.

Reply:
👍 to tidying this up - these are mostly unnecessary...

> We don't use curly brackets in switch statements anywhere else in the app (that I know of)

For extra context, I know of at least these two places where we've needed to do this:

https://github.com/desktop/desktop/blob/82c188d68a83330a08feacec782e32fee3eed5a4/app/src/ui/app.tsx#L1560-L1586

https://github.com/desktop/desktop/blob/82c188d68a83330a08feacec782e32fee3eed5a4/app/src/ui/app.tsx#L1612-L1659

This is because local variables inside a `case` share the same scope. If you remove the curly brackets here you'll see the error that `tsc` complains about:

<img width="695"  src="https://user-images.githubusercontent.com/359239/58428434-d5a06400-8078-11e9-922e-08f661089288.png">
<img width="678"  src="https://user-images.githubusercontent.com/359239/58428435-d5a06400-8078-11e9-88c3-7f742a9d2a1c.png">